### DISCUSSION TITLE
where to set custom config in milvus v2.x?

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15958] [Q&A and General discussion] [answered]
Hi, there used to be server_config.yml  file for milvus serving before version2.0. server_config.yml provides many custom configs values, which is disappeared in version 2.x.

I am wondering how can I set config value for milvus v2.x system as described in wiki: https://milvus.io/cn/blog/2020-2-19-milvus-config.md

(for standalone and cluster mode)

**Comment by yhmo** [comment]
Please read this document: https://milvus.io/docs/v2.0.x/configure-docker.md

**Reply by xiaojuan8** [reply]
thanks！

**Reply by xiaojuan8** [reply]
hi, btw, another question, no "cache.cache_size" key word in milvus.yaml, where should I set this config for milvus searching? @yhmo

**Reply by yhmo** [reply]
In the milvus.yaml, there is a queryNode.cacheSize:

```
queryNode:
  cacheSize: 32 # GB, default 32 GB, `cacheSize` is the memory used for caching data for faster query. The `cacheSize` must be less than system memory size.
```

**Reply by xiaojuan8** [reply]
thanks!

**Reply by xiaojuan8** [reply]
hi, I found that milvus v2.0 has different configuration for docker-compose and helm respectively, 
docker version: https://raw.githubusercontent.com/milvus-io/milvus/v2.0.1/configs/milvus.yaml 
helm version : https://raw.githubusercontent.com/milvus-io/milvus-helm/master/charts/milvus/values.yaml

do they share same config parameters, I mean if I cannot find cacheSize under queryNode in values.yaml, I can directly add this key to values.yaml under queryNode, and apply value.yaml with helm? Is it true? @yhmo

**Reply by yhmo** [reply]
The milvus-helm project didn't expose the item. Please go to the milvus-helm project to create an issue: https://github.com/milvus-io/milvus-helm

**Comment by LoveEachDay** [comment, marked as answer]
If you deploy with milvus-helm chart, you can set the queryNode.cacheSize to 8GB like this in the values.yaml to :
```
queryNode:
   resources:
      limits:
         memory: "8Gi"
```

**Reply by xiaojuan8** [reply]
great, thanks!!
